src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java

```diff
@@ -106,30 +106,31 @@ protected PreviewStatus doInBackground() throws Exception {
             return new PreviewStatus(State.CANCELLED, 0, previewLimit / 1024, KB, name, EMPTY_BYTE_ARRAY);
           }
         }
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream(previewLimit);
-        InputStream inputStream = null;
+        ByteArrayOutputStream outputStreamRef = null;
```

Comment:
Neede due to the scope of try-with-resources as used later for more general exception